tools/RAiDER/aria/prepFromARIA.py

```diff
@@ -0,0 +1,212 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Author: Jeremy Maurer, Brett Buzzanga
+# Copyright 2022, by the California Institute of Technology. ALL RIGHTS
+# RESERVED. United States Government Sponsorship acknowledged.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+import os
+from datetime import datetime
+import numpy as np
+import xarray as xr
+import rasterio
+import yaml
+import RAiDER
+from RAiDER.utilFcns import rio_open, writeArrayToRaster
+from RAiDER.logger import logger
+
+""" Should be refactored into a class that takes filename as input """
+## ---------------------------------------------------- Prepare Input from GUNW
+def makeLatLonGrid(f:str, out_dir:str):
+# def makeLatLonGrid(f:str, reg, out_dir):
+    ds0  = xr.open_dataset(f, group='science/grids/data')
+
+    Lat, Lon  = np.meshgrid(ds0.latitude.data, ds0.longitude.data)
+
+    da_lat = xr.DataArray(Lat.T, coords=[Lon[0, :], Lat[:, 0]], dims='lon lat'.split())
+    da_lon = xr.DataArray(Lon.T, coords=[Lon[0, :], Lat[:, 0]], dims='lon lat'.split())
+    # dst_lat = op.join(out_dir, f'lat_{reg}.geo')
+    # dst_lon = op.join(out_dir, f'lon_{reg}.geo')
+
+    dst_lat = os.path.join(out_dir, 'lat.geo')
+    dst_lon = os.path.join(out_dir, 'lon.geo')
+    da_lat.to_netcdf(dst_lat)
+    da_lon.to_netcdf(dst_lon)
+    logger.debug('Wrote: %s', dst_lat)
+    logger.debug('Wrote: %s', dst_lon)
+    return dst_lat, dst_lon
+
+
+def getHeights(f:str):
+    ds =  xr.open_dataset(f, group='science/grids/imagingGeometry')
+    hgts = ds.heightsMeta.data.tolist()
+    return hgts
```

Comment:
Should be a configurable parameter in the future. 
The heights needs to span the earths topographic range [min and max topography]
The heights steps should be comparable to that of the model.

2 Caveats here:
- if we want to be able to stitch in ARIA-tools across products the heigh-levels needs to be consistent. Moreover the grids needs to be consistent and integer multiple. 
- If we now consider swapping between models e.g. HRES and HRRR we should likely also have the heights consistent (should be ok as they are similar models - TBC) and the lateral should be a multiple grid increment (TBC if that can be combined in ARIA-tools to). tagging @piyushrpt @sssangha

Reply:
@bbuzz31 @dbekaert are we thinking to be able to add delay as a cube to the product or only ever the interferometric delays? If the latter I think the better way to get heights would to use the raider default in which the weather model height levels directly.

Reply:
I think we should move this discussion to #441 and just leave this branch as implementing the CLI changes

Reply:
We should define our own grid and heights; per my other comment so we can ensure use of the models and mixing along product edges and boundaries.